README.md

```diff
@@ -1,30 +1,145 @@
+toolchain for your target platform. Examples of how we cross-compile rav1d in CI
+can be found in `.github/workflows/build-and-test-qemu.yml`.
 
 ## Running Tests
 
-Currently we use the original Meson test suite for testing the Rust port. To setup and run these tests, do the following:
+Currently we use the original [Meson](https://mesonbuild.com/) test suite for
+testing the Rust port. This means you'll need to [have Meson
+installed](https://mesonbuild.com/Getting-meson.html) to run tests.
+
+To setup and run the tests, do the following:
 
-First, build the Rust project using Cargo. You'll need to do this step manually before running any tests because it is not built automatically when tests are run. Note that we build with the `--release` flag, adjust paths accordingly
-for debug or cross-target builds.
+First, build the Rust project using Cargo. You'll need to do this step manually
+before running any tests because it is not built automatically when tests are
+run. It's recommended to run tests with either the release or opt-dev profile as
+the debug build runs slowly and often causes tests to timeout. The opt-dev
+profile is generally ideal for development purposes as it enables some
+optimizations while leaving debug checks enabled.
 
-```txt
+```sh
 cargo build --release
 ```
 
-Second, create the `build` dir with `meson`:
+Or:
 
-```txt
-meson setup build
+```sh
+cargo build --profile opt-dev
 ```
 
-Then you can run the tests with:
+Then you can run the tests with the `tests.sh` helper script:
 
-```txt
+```sh
 .github/workflows/test.sh -r target/release/dav1d
 ```
 
-You can learn more about how to build and test in the `.github/workflows` folder.
+Or:
+
+```sh
+.github/workflows/test.sh -r target/opt-dev/dav1d
+```
+
+The test script accepts additional arguments to configure how tests are run:
+
+* `-s PATH` - Specify a path to the seek-stress binary in order to run the
+  seek-stress tests. This is generally in the same output directory as the main
+  `dav1d` binary, e.g. `target/release/seek_stress`.
+* `-t MULTIPLIER` - Specify a multiplier for the test timeout. Allows for tests
+  to take longer to run, e.g. if running tests with a debug build.
+* `-f DELAY` - Specify a frame delay for the tests. If specified the tests will
+  also be run with multiple threads.
+* `-n` - Test with negative strides.
+* `-w WRAPPER` - Specify a wrapper binary to use to run the tests. This is
+  necessary for testing under QEMU for platforms other than the host platform.
+
+You can learn more about how to build and test by referencing the CI scripts in
+the `.github/workflows` folder.
+
+# Using rav1d
+
+rav1d is designed to be a drop-in replacement for dav1d, so it primarily exposes
+a C API with the same usage as dav1d's. This is found in the `librav1d.a`
+library generated by `cargo build`. dav1d's primary API documentation [can be
+found here](https://videolan.videolan.me/dav1d/dav1d_8h.html) for reference, and
```

Comment:
```suggestion
library generated by `cargo build`. dav1d's [primary API documentation can be
found here](https://videolan.videolan.me/dav1d/dav1d_8h.html) for reference, and
```

It's important to include what the link is about in the link text for screen readers.  For example, "can be found here" isn't a very useful description.